day_7/ceaser_cipher.rb

```diff
@@ -0,0 +1,17 @@
+ALPHABET_SIZE = 26
+
+def caesar_cipher(string)
+  shiftArray = []
+  charLine = string.chars.map(&:ord)
+
+shift = 1
+ALPHABET_SIZE.times do |shift|
+  shiftArray << charLine.map do |c|
+    ((c + shift) < 123 ? (c + shift) : (c + shift)- 26).chr
+  end.join
+end
+
+  shiftArray
+end
+
+puts caesar_cipher("Grinch")
```

Comment:
This is clearly a copy-paste of [someone else's code](https://gist.github.com/matugm/db363c7131e6af27716c). Delete and replace with your own pseudocode in comments that explain each step so you can code the solution and fully understand it.

Reply:
sorry i was trying to understand what the code was doing and work it out line by line and must have had to stop. i will work on these today.